docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
           memory: 128M
     env_file:
       - ./.env
+      - ./.env.local
```

Comment:
it should be good to have it in the documentation as an extra step in order to have a fully working environment

Reply:
there is already: https://github.com/PUGX/badge-poser/blob/master/CONTRIBUTING.md#configure-your-envlocal-file